modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm

```diff
@@ -152,3 +152,19 @@
 	new /obj/item/storage/photo_album/personal(src)
 	new /obj/item/bedsheet/centcom(src)
 	new /obj/item/storage/bag/garment/nanotrasen_consultant(src)
+
+//Choice Beacon, I hope in the future they're going to be given proper unique gun but this will do.
+
+
+/obj/item/choice_beacon/ntc //I promise you, this will solve everything
```

Comment:
Also, maybe chill out with the weird commentary on the sides of this, it just adds more commentary that doesn't serve any purpose.